### PR TITLE
MGMT-9388: Incorrect minimal CPU and RAM requirements for cnv operator shown in UI

### PR DESCRIPTION
**Pull request by tsorya**
[MGMT-9388](https://issues.redhat.com//browse/MGMT-9388): Incorrect minimal CPU and RAM requirements for cnv operator shown in UI
Changing cnv minimum requirements for sno (master)

## List all the issues related to this PR

- [ ] New Feature 
- [ ] Enhancement 
- [x] Bug fix
- [ ] Tests
- [ ] Documentation
- [ ] CI/CD 

## What environments does this code impact?

- [ ] Automation (CI, tools, etc)
- [x] Cloud
- [ ] Operator Managed Deployments
- [] None

## How was this code tested?

- [ ] assisted-test-infra environment
- [ ] dev-scripts environment
- [ ] Reviewer's test appreciated
- [ ] Waiting for CI to do a full test run
- [x] Manual (Elaborate on how it was tested)
- [] No tests needed

## Assignees

/cc @
/cc @

## Checklist

- [ ] Title and description added to both, commit and PR.
- [ ] Relevant issues have been associated (see [CONTRIBUTING] guide)
- [ ] Reviewers have been listed
- [ ] This change does not require a documentation update (docstring, `docs`, README, etc)
- [ ] Does this change include unit-tests (note that code changes require unit-tests)

## Reviewers Checklist

- Are the title and description (in both PR and commit) meaningful and clear?
- Is there a bug required (and linked) for this change?
- Should this PR be backported?

[Kubernetes community documentation]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines
[CONTRIBUTING]: https://github.com/openshift/assisted-service/blob/master/CONTRIBUTING.md

### CODE REVIEW

File: internal/operators/cnv/cnv_operator.go [hunk 1]
```diff
@@ -230,6 +229,12 @@ func (o *operator) GetPreflightRequirements(_ context.Context, cluster *common.C
 			},
 		},
 	}
+
+	if common.IsSingleNodeCluster(cluster) {
+		requirements.Requirements.Master.Quantitative.CPUCores += WorkerCPU
+		requirements.Requirements.Master.Quantitative.RAMMib += WorkerMemory
```
Comment (eranco74) [inline internal/operators/cnv/cnv_operator.go:235]: Why?

Comment (tsorya) [reply internal/operators/cnv/cnv_operator.go:235]: cause SNO is actually master and worker and in validation we check it though in UI we showed only master requirements without adding worker's

**Pull request merged**